srcpkgs/chirp/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'chirp'
+pkgname=chirp
+version=20190220
+revision=1
+wrksrc="chirp-daily-${version}"
+build_style=python2-module
```

Comment:
Sorry but i'm not particularly happy about adding packages for a close to EOL python version

Reply:
seconded. Upstream is working towards python3 support so let's just wait until it is ready and then add it into the repository.